com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 ### Fixed
 - Fixed a bug that was converting previous Alembic instance in prefabs into out-of-project references.
+- Fixed a bug that was polluting the Undo stack while assets were being imported.
```

Comment:
Suggestion:
`- Fixed a bug that was adding unnecessary Undo events during the Alembic asset import process.`